.jenkins/Jenkinsfile.cypress

```diff
@@ -0,0 +1,24 @@
+// Example Jenkins pipeline with Cypress
+pipeline {
+  agent {
+     docker {
+       image 'saucelabs/stt-cypress-node:latest'
```

Comment:
Wrong name :D --> https://hub.docker.com/r/saucelabs/stt-cypress-mocha-node
```suggestion
       image 'saucelabs/stt-cypress-mocha-node:latest'
```

Reply:
Oh, I'll update that. I just copied from circelci config. Can I update that image name in circleci config and github config as well?